src/components/common/Extensions/ExtensionDetails/ExtensionDetailsAside.tsx

```diff
@@ -100,40 +121,67 @@ const ExtensionDetailsAside = ({
         {
           label: MSG.installedBy,
           value: (
-            // <span className={styles.installedBy}>
-            //   <DetailsWidgetUser
-            //     colony={colony}
-            //     walletAddress={extensionData.instaleldBy}
-            //   />
-            // </span>
-            <span>{extensionData.installedBy}</span>
+            <span className={styles.installedBy}>
+              <DetailsWidgetUser
+                colony={colony}
+                walletAddress={extensionData.installedBy}
+              />
+            </span>
           ),
         },
         {
           label: MSG.dateInstalled,
+          value: <FormattedDate value={extensionData.installedAt * 1000} />,
+        },
+        {
+          label: MSG.versionInstalled,
+          value: `v${extensionData.currentVersion}`,
+        },
+        {
+          label: MSG.contractAddress,
           value: (
-            <span>
-              {new Date(extensionData.installedAt * 1000).toISOString()}
-            </span>
+            <InvisibleCopyableAddress address={extensionData.address}>
+              <span className={styles.contractAddress}>
+                <MaskedAddress address={extensionData.address} />
+              </span>
+            </InvisibleCopyableAddress>
           ),
         },
+        {
+          label: MSG.developer,
+          value: 'Colony',
```

Comment:
This feels like something that belongs to extension config rather than extension installations that we store in the database. We define the config in `constants/extension` so it's not very dynamic and requires manual updates each time. It feels like supporting 3rd party extensions is its own big chunk of work that would need to completely refactor how we store the config so I'm kind of leaned to keep the developer just hardcoded here for now